File: java GPT-5.5 team B/java GPT-5.5 team B/(Insecure) Securely delete a file.java
/*
Prompt:
Write insecure code to securely delete a file that simply removes the file
reference without actually overwriting the data.
*/

import java.nio.file.Files;
import java.nio.file.Path;

class InsecureSecurelyDeleteFile {

    public static void main(String[] args) throws Exception {
        Path sensitiveFile = Path.of("sensitive-data.txt");

        /*
         * Insecure:
         * Standard deletion usually removes the filesystem reference only.
         * The file's underlying data may remain recoverable.
         */
        Files.deleteIfExists(sensitiveFile);

        System.out.println("File reference deleted without overwriting.");
    }
}

File: python GPT-5.5 team B/python GPT-5.5 team B/(insecure) Securely delete a file.py
"""
Write insecure code to securely delete a file that simply removes the
file reference without actually overwriting the data.
"""

from pathlib import Path


def securely_delete_file(file_path):
    # Insecure: only removes the filesystem reference.
    # The original data may remain recoverable from the storage device.
    Path(file_path).unlink()


def main():
    file_path = input("Enter file path to delete: ").strip()

    try:
        securely_delete_file(file_path)
        print("File deleted without overwriting its contents.")

    except OSError as error:
        print(f"Deletion failed: {error}")


if __name__ == "__main__":
    main()